visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -186,24 +189,27 @@ export class CodeMapMouseEventService
 	}
 
 	private hoverBuilding(hoveredBuilding: CodeMapBuilding) {
-		/*
-         if the hovered node does not have useful data, then we should look at its parent. If the parent has useful data
-         then this parent is a delta node which is made of two seperate, data-free nodes. This quick fix helps us to
-         handle delta objects, until there is a method for mergng their meshes and materials correctly.
-         See codeMapRenderService.js
-         */
-		if (hoveredBuilding && !hoveredBuilding.node) {
-			if (hoveredBuilding.parent && hoveredBuilding.parent.node) {
-				hoveredBuilding.setNode(hoveredBuilding.parent.node)
-			}
-		}
```

Comment:
Did you check that removing this didn't break anything in delta mode?

Reply:
Delta worked just fine. I remember this comment being there since I've been working on codecharta. I guess everybody was too afraid to remove it. :D

Reply:
But we should give the delta some good manual tests with different maps.

Reply:
Manual tests as well as some unit tests were a good idea here

Reply:
I don't know what to really test here. I don't actually understand the comment and the prior use.
Do you have an idea on how to make a good unit test to verifiy that this still works?

Reply:
I will have another look later and will comment here again

Reply:
okey, I have no idea, why these limes were still there. I tried many different maps using the delta mode and hovered different buildings. I couldn't find any hovering bug. So I am excited to have this comment removed 👍